### PR TITLE
Add "Follows You" text to user header for users that follow current auth user

### LINKED ISSUE #964: [Feature Proposal] "Follows You" indicator
Per our new development guidelines, I wanted to have discussion / get approval to begin working on a feature I proposed on Utopian.

You can read the full post here: [Utopian: Busy Feature Idea: "Follows You" indicator](https://utopian.io/utopian-io/@ryanbaer/busy-feature-idea-follows-you-indicator)

But here are some screenshots which convey what I had in mind:
image

image

### PR DESCRIPTION
**Pull request by jm90m**
Add "Follows You" text to user header for users that follow current auth user

Fixes #964

Changes:
- Optimize follows your request made
- display "Follows you" if you use follows you

### CODE REVIEW

File: src/client/locales/en.json [hunk 1]
```diff
@@ -298,5 +298,6 @@
   "login_modal_footer_text": "Don't have an account? Signup with {link}",
   "login_to_busy": "Login to Busy",
   "login_modal_description": "Login with your Steem account using SteemConnect to enjoy Busy at 100%",
-  "login_with_steemconnect": "Login with SteemConnect"
-}
\ No newline at end of file
+  "login_with_steemconnect": "Login with SteemConnect",
+  "follows_you": "Follows you"
```
Comment (Sekhmet) [inline src/client/locales/en.json]: What do we do with translations at the moment?
Should they be in en.json or default.json?

@bonustrack

Comment (bonustrack) [reply src/client/locales/en.json]: Translation should be in `default.json`

Comment (bonustrack) [reply src/client/locales/en.json]: Good catch, `en.json` should not be used anymore.

Comment (jm90m) [reply src/client/locales/en.json]: should it be in both en.json and default.json? @bonustrack @Sekhmet

Comment (bonustrack) [reply src/client/locales/en.json]: For now yes @jm90m

**Pull request merged**